lms/djangoapps/bookmarks/views.py

```diff
@@ -166,6 +167,19 @@ def get_queryset(self):
 
         return api.get_bookmarks(user=self.request.user, course_key=course_key, serialized=False)
 
+    def paginate_queryset(self, queryset, page_size=None):
+        """ Override GenericAPIView.paginate_queryset for the purpose of eventing """
+        page = super(BookmarksListView, self).paginate_queryset(queryset, page_size)
+        tracker.emit(
+            'edx.course.bookmark.listed',
+            {
+                'bookmarks_count': page.paginator.count,
+                'page_size': self.get_paginate_by(),
+                'page_number': page.number,
```

Comment:
Can you add a test which tests different data to make sure this values are always as expected?